CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Release History
-## 1.0.0 (2012-11-01)
+
+## 1.1.0 (2023-01-19)
+
+### Behavior Changes
+- `SnowparkSourceCodeExtractionException` is raised when Snowpark fails to extract the source code of a function referenced by UDF registration.
+-  In `Session.create_dataframe()`, if `data` is a pandas DataFrame and the optional parameter `schema` is specified, `Session.create_dataframe()` now uses `schema` as the resulting DataFrame's schema.
+
+### New Features:
+- Added `asc`, `asc_nulls_first`, `asc_nulls_last`, `desc`, `desc_nulls_first`, `desc_nulls_last`, `date_part` and `unix_timestamp` in functions.
+- Enabled creating UDFs that reference non-local functions. This feature is not available inside stored procedures.
+- Added the property `DataFrame.dtypes` to return a list of column name and data type pairs.
+- Added the following aliases:
+  - `functions.expr()` for `functions.sql_expr()`.
+  - `functions.date_format()` for `functions.to_date()`.
+  - `functions.monotonically_increasing_id()` for `functions.seq8()`
+  - `functions.from_unixtime()` for `functions.to_timestamp()`
+
+### Bug Fixes:
+- Fixed issue https://github.com/snowflakedb/snowpark-python/issues/658.
```

Comment:
I suggest we keep the original description and have a reference to the issue.

Reply:
@sfc-gh-mabrennan Do you mind if we put an issue URL in the release notes?

Reply:
Go ahead.